src/main/java/uk/gov/hmcts/reform/cpo/service/impl/CasePaymentOrdersServiceImpl.java

```diff
@@ -47,6 +50,12 @@ public CasePaymentOrdersServiceImpl(CasePaymentOrdersRepository casePaymentOrder
         }
     }
 
+    @Transactional
+    @Override
+    public CasePaymentOrder updateCasePaymentOrder(UpdateCasePaymentOrderRequest request) {
+        throw new UnsupportedOperationException("Implement me: see CPO-6");
+    }
+
     private PageRequest getPageRequest(CasePaymentOrderQueryFilter casePaymentOrderQueryFilter) {
         final List<Sort.Order> orders = new ArrayList<>();
```

Comment:
getPageReques() has been moved to CasePaymentOrderQueryFilter, based on suggestions from reviewer.

Reply:
Thanks no action has been made in this PR.